pandas/core/internals/managers.py

```diff
@@ -727,15 +727,7 @@ def copy(self, deep: bool | Literal["all"] = True) -> Self:
         BlockManager
         """
         # this preserves the notion of view copying of axes
-        if deep:
-            # hit in e.g. tests.io.json.test_pandas
-
-            def copy_func(ax):
-                return ax.copy(deep=True) if deep == "all" else ax.view()
-
-            new_axes = [copy_func(ax) for ax in self.axes]
-        else:
-            new_axes = [ax.view() for ax in self.axes]
+        new_axes = [ax.view() for ax in self.axes]
```

Comment:
Reason that this is removed?

I know that an Index is generally immutable so the difference between view or actual copy is not relevant when it comes to the _data_, but we still some mutable _attributes_